test/unit/effects.js

```diff
@@ -13,20 +13,50 @@ var oldRaf = window.requestAnimationFrame,
 
 QUnit.module( "effects", {
 	setup: function() {
-		window.requestAnimationFrame = null;
 		this.sandbox = sinon.sandbox.create();
 		this.clock = this.sandbox.useFakeTimers( 505877050 );
+
 		this._oldInterval = jQuery.fx.interval;
+		this._oldPerformanceNow = window.performance.now;
 		jQuery.fx.step = {};
 		jQuery.fx.interval = 10;
+
+		// Simulate rAF using the mocked setTimeout as otherwise we couldn't test
+		// the rAF + performance.now code path.
+		// Support: IE <10, Android <4.4, Node with jsdom 9.4
+		if ( window.requestAnimationFrame ) {
+			window.requestAnimationFrame = function( callback ) {
```

Comment:
Don't know, never used one before :), I'm thinking we should be aware of the all options before creating our own, that kinda thing